procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb

```diff
@@ -22,13 +22,17 @@
     "import time\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        time.sleep(10)
</pre><p>So first any sleeps should use <code>await asyncio.sleep(time_in_seconds)</code></p><p>However, see the example above for the bias? I think you want to catch the ingestion event instead of sleep </p><p><code>&nbsp;&nbsp;ingest_event = await latiss.rem.atoods.evt_imageInOODS.next(flush=False, timeout=10)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Yes, thanks for the clarification!</p>